content/en/docs/features/encrypted-images.md

```diff
@@ -86,22 +93,44 @@ skopeo copy dir:./oci/output "docker://${ENCRYPTED_IMAGE}"
 Prior to launching a Pod the image key needs to be provisioned to the Key Broker's repository. For a KBS deployment on Kubernetes using the local filesystem as repository storage it would work like this:
 
 ```bash
-kubectl exec deploy/kbs -- mkdir -p "/opt/confidential-containers/kbs/repository/$(dirname "$KEY_PATH")"
-cat "$KEY_FILE" | kubectl exec -i deploy/kbs -- tee "/opt/confidential-containers/kbs/repository/${KEY_PATH}" > /dev/null
+kubectl exec deploy/kbs -n coco-tenant -- mkdir -p "/opt/confidential-containers/kbs/repository/$(dirname "$KEY_PATH")"
```

Comment:
Where does the coco-tenant runtime class come from?

Reply:
It's not runtime class but default namespace which is used for KBS (coco-tenant) - in documentation (I checked couple other pages) also it's used with namespace and as far as I know KBS is never deployed with default namespace

Reply:
This namespace is in the yamls provided in the Trustee repo, but it's not in the trustee-operator repo. Do we still use it when installing via the operator? We can probably merge this either way, but we may have an inconsistency here.

Reply:
This `coco-tenant` namespace is causing issues when interacting with kbs.

```
$ kubectl exec deploy/kbs -n coco-tenant -- mkdir -p "/opt/confidential-containers/kbs/repository/$(dirname "$KEY_PATH")"
Error from server (NotFound): namespaces "coco-tenant" not found
```

Reply:
Ok, we've yet to provide good instructions for setting up attestation in the docs. I picture offloading the specifics of deploying and provisioning the KBS to other pages where we could address this sort of thing. For now, maybe make a note idk.

Reply:
I'll remove coco-tenant and replace it with note about it